arm-powerbiembedded/2016-01-29/swagger/powerbiembedded.json

```diff
@@ -338,7 +331,7 @@
         "tags": [
           "WorkspaceCollections"
         ],
-        "operationId": "WorkspaceCollections_listBySubscription",
+        "operationId": "WorkspaceCollections_list",
```

Comment:
@veronicagg [unfortunately AutoRest is complaining about the current name at generation time](https://cloud.githubusercontent.com/assets/666005/24123380/8c3785f6-0dbf-11e7-9656-82c426cffbdc.png) - so it's going to be a breaking change regardless of what's picked, but sure, I can update this to be `listBySubscriptionID` :)